### PR TITLE
fix: Correct `fargate-fluentbit` example usage

### LINKED ISSUE #1076: Cannot Create Log Stream due to Double Quote
https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/0848d87e4ab52f76dfff2e3d0c6d64d196b6e877/examples/complete-kubernetes-addons/main.tf#L189

The double quotation marks in the log_stream_prefix field in the example of the Fargate Fluentbit addon causing the SerializationException during the log stream creation. It works fine if the double quotation marks are removed.

### PR DESCRIPTION
**Pull request by bryantbiggs**
### What does this PR do?

- Remove self-managed node group and Fargate profile from `complete-kubernetes-addons`
- Remove `fargate-fluentbit` example usage; this is shown in the `fargate-serverless` example

### Motivation

- Resolves #1076

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs require a new example and/or doc page. In general:
- Use an existing example when possible to demonstrate a new addons usage
- A new docs page under `docs/add-ons/*` is required for new a new addon

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/complete-kubernetes-addons/main.tf [hunk 1]
```diff
@@ -96,35 +96,6 @@ module "eks_blueprints" {
     }
   }
 
-  self_managed_node_groups = {
```
Comment (bryantbiggs) [inline examples/complete-kubernetes-addons/main.tf:99]: not necessary to have all 3 compute types here for "complete addons"

File: examples/complete-kubernetes-addons/main.tf [hunk 2]
```diff
@@ -159,13 +130,6 @@ module "eks_blueprints_kubernetes_addons" {
   enable_aws_for_fluentbit                 = true
   aws_for_fluentbit_cw_log_group_retention = 30
   aws_for_fluentbit_helm_config = {
-    name                            = "aws-for-fluent-bit"
```
Comment (bryantbiggs) [inline examples/complete-kubernetes-addons/main.tf:162]: just showing the bits that are custom

File: examples/complete-kubernetes-addons/main.tf [hunk 3]
```diff
@@ -178,41 +142,6 @@ module "eks_blueprints_kubernetes_addons" {
     ]
   }
 
-  enable_fargate_fluentbit = true
-  fargate_fluentbit_addon_config = {
```
Comment (bryantbiggs) [inline examples/complete-kubernetes-addons/main.tf:182]: this is nearly (99.9%) identical to the defaults so removing entirely. usage of this addon is still shown though in the `fargate-serverless` example

File: examples/complete-kubernetes-addons/main.tf [hunk 4]
```diff
@@ -268,12 +197,3 @@ module "vpc" {
 
   tags = local.tags
 }
-
-data "aws_ami" "eks" {
```
Comment (bryantbiggs) [inline examples/complete-kubernetes-addons/main.tf:272]: was used on self-managed node group thats removed

File: examples/fargate-serverless/main.tf [hunk 5]
```diff
@@ -121,6 +121,9 @@ module "eks_blueprints_kubernetes_addons" {
 
   # Enable Fargate logging
   enable_fargate_fluentbit = true
+  fargate_fluentbit_addon_config = {
+    flb_log_cw = true
```
Comment (bryantbiggs) [inline examples/fargate-serverless/main.tf:125]: from recent PR #1129

**Review by Zvikan — APPROVED** [review]

**Review by askulkarni2 — APPROVED** [review]
Lgtm!

**Pull request merged**